lib/errors.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+/**
+ * @module Errors
+ */
+/**
+ * Factory functions to create throwable error objects
+ */
+
+/**
+ * File/s of test could not be found.
+ *
+ * @public
```

Comment:
Incomplete documentation on each function...
```js
  * @param {string} message - Error message to be displayed.
  * @param {string} pattern - User-specified argument value.
  * @returns {Error} instance detailing the error condition 
```

Reply:
Will update. Do you mean `@public` here? They are exported so my assumption would be `@public`?

Reply:
Only exported variables are public. You roundabout export your functions via the `module.exports` object whose properties you assign them to, but those properties are the _real_ `@public`s. But I could see documenting these as such... your call.